src/ext/java/org/opentripplanner/ext/airquality/configuration/IndexVariable.java

```diff
@@ -0,0 +1,64 @@
+package org.opentripplanner.ext.airquality.configuration;
```

Comment:
What is your chosen name of the sandbox feature? The feature toggle is called `DataOverlay` but the package is called `airquality`.